apps/condo/domains/analytics/schema/GetOverviewDashboardService.test.js

```diff
@@ -109,11 +108,13 @@ describe('GetOverviewDashboardService', () => {
         ticket = createdTicket
     })
 
+    beforeEach(() => {
```

Comment:
It's a bit strange to see `beforeEach` at the end of `describe` block :)

Reply:
Also should feature flag be toggled down in `afterEach`?

Reply:
In most of the test cases feature flag should be enabled. But in a single test we should check if value of the `ANALYTICS_V3` flag is `false`, error was provided to the output.

From the other side, there is a case when developer want to run single test (for example with error about disabled feature). So the easiest way to make this work is enable feature before each test and disable feature if necessary